### DISCUSSION TITLE
Per bulb brightness settings via UI?

### DISCUSSION THREAD

**Discussion by WhatMeansGithub** [basnijholt/adaptive-lighting#195] [Q&A]
I've recently started diversifying my bulbs and have different types with different brightness outputs. However, I can't seem to figure out how to adjust brightness per bulb. For example, I have a 1000 lumen bulb in place of an older 800 lumen bulb. The brightness output on the 800 lumen bulb was perfect, the 1000 lumen bulb is too bright for daily use and I only need its full power in some situations. Would it be somehow possible to have that light come on at 80% and then dim accordingly along with my other bulbs?

**Comment by Zipties** [comment]
I've been wanting a way to do this, but for kelvin. Some of my bulbs are slightly warmer or cooler than others and being able to calibrate every bulb would be really nice
